draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -340,37 +354,59 @@
           configured with an algorithm to combine the recognized metrics
           carried by the Sub-TLVs within a Metadata Path Attribute of the
           received BGP UPDATE message.</t>
+	  <!-- XXX JMH - This text is really an error handling consideration.
+	  Perhaps consider renaming the section to note that? -->
         </section>
 
         <section title="Metadata Path Attribute Handling Procedure">
           <t>The Metadata Path Attribute MUST contain at least one metadata
           Sub-TLV. Multiple Metadata Sub-TLVs can be included in a Metadata
           Path Attribute in one BGP UPDATE message. The content of the
-          Sub-TLVs present in the BGP Metadata Path attribute is determined by
-          the configuration. When a BGP Speaker does not recognize some of the
-          Sub-TLVs within one Metadata Path Attribute in a BGP UPDATE message,
-          the BGP Speaker should forward the received BGP UPDATE message
-          without any change if the transitive bit is set to 1 <xref
-          target="RFC4271"/>. The domain ingress nodes should process the
+          Sub-TLVs present in the BGP Metadata Path Attribute is determined by
+          configuration.  The domain ingress nodes should process the
           recognized Sub-TLVs carried by the Metadata Path Attribute and
           ignore the unrecognized Sub-TLVs. By default, a BGP speaker does not
           report any unrecognized Sub-TLVs within a Metadata Path Attribute
           unless configured to send a notification to its management system.
           The ingress node should be configured with an algorithm to combine
           the recognized metrics carried by the Sub-TLVs within a Metadata
           Path Attribute of the received BGP UPDATE message.</t>
+	  <!-- XXX JMH "combine" is odd.  Perhaps utilize? -->
+
```

Comment:
how about "consider" (instead of "combine")?

Reply:
That works.